.circleci/config.yml

```diff
@@ -119,6 +119,7 @@ commands:
     steps:
       - run: sudo apt-get update
       - run: sudo apt-get install python tcl
+      - run: sudo apt-get install python3-venv
```

Comment:
Good call, I didn't think about updating the development setup docs. In this case, if they have python installed they _should_ have venv installed. This was only necessary because the Debian images we use in CI don't set a flag (ensurepip) which means that you have to explicitly install these packages.